deploy/packaging/alpine/build_apks.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+pckdir=$(dirname $(realpath $0))
+cd /workspace
+signkeys="/sign_keys/mdsplus@mdsplus.org-589e05b6.rsa"
```

Comment:
If we're hardcoding the path to the key here, why don't we just download it here as well?
The Dockerfile calls wget to download the key, we could instead add a check here for the file and download it if it doesn't exist, that way there wouldn't be any hardcoded file paths between the image and the build scripts

Reply:
you cannot instlal the key wiothout sudo rigjhts and in jekins you are not necessarely sudo